src/core/domain/entities/dummy.entity.ts

```diff
@@ -7,12 +7,17 @@
  * @return void
  *
  * @description La estructura basicamente es lo que esta en el MER,
- * usar tipos de datos primitivos (number, string, boolean, etc)
+ * se tiene la idea usar tipos de datos personalizados, como UUID.
  */
 
-export class Dummy {
-  constructor(
-    public id: string,
-    public name: string
-  ) {}
+export interface Dummy {
+  /**
+   * @param id {string} - Identificador de la entidad se cambia a UUID
```

Comment:
Creo que como es typescript es mala práctica definir el tipo de dato en el comment.

Reply:
No es mala practica, simplemente es preferencia del autor del código y creo que deberíamos de definir esa parte del estándar para nuestro código. Doxygen si te permite poner los tipos en formato de JSDocs o TS style. 

1. Con {type} así
/**
 * @param {string} paramName Description of the parameter
 */

2. Con sintaxis de TS
/**
 * @param paramName: string Description of the parameter
 */

Al final depende mucho de la preferencia como equipo, porque si puede ser más trabajo estar escribiendo tipos o reescribiendo si hay cambios (considerando que ya se especifican en el código), pero no le veo lo malo de escribir el tipo dentro del comentario.

Reply:
`@param paramName: string Description of the parameter`

esta me parece una excelente opcion